src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -6,78 +6,84 @@
 import static org.folio.circulation.support.Result.succeeded;
 import static org.folio.circulation.support.ValidationErrorFailure.failedValidation;
 
+import java.lang.invoke.MethodHandles;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
 
 import org.folio.circulation.domain.policy.RequestPolicy;
 import org.folio.circulation.domain.policy.RequestPolicyRepository;
 import org.folio.circulation.support.ResponseWritableResult;
 import org.folio.circulation.support.Result;
 import org.folio.circulation.support.http.server.ValidationError;
 
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
 public class CreateRequestService {
+
+  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
   private final RequestRepository requestRepository;
   private final UpdateItem updateItem;
   private final UpdateLoanActionHistory updateLoanActionHistory;
   private final RequestPolicyRepository requestPolicyRepository;
+  private final LoanRepository loanRepository;
   private final UpdateLoan updateLoan;
 
-  public CreateRequestService(
-    RequestRepository requestRepository,
-    UpdateItem updateItem,
-    UpdateLoanActionHistory updateLoanActionHistory,
-    UpdateLoan updateLoan,
-    RequestPolicyRepository requestPolicyRepository) {
+  public CreateRequestService(RequestRepository requestRepository, UpdateItem updateItem,
```

Comment:
At some point I reckon we need to agree what our line separation policy is. Your change is more in line with the guidelines.